### ISSUE TITLE
Reimplement the finetune_scan_objonly.yaml, only 86.78 acc, why?

### ISSUE THREAD

**Issue by cuge1995** [Pang-Yatian/Point-MAE#6]

**Comment by Pang-Yatian** [comment]
Hi,

We have revised the reported result of scan_objonly and scan-objbg. Scan_objonly acc is 88.29% and scan-objbg acc is 90.02%.

We double-check the experiments, and can easily get reported results. See new log [objonly](https://drive.google.com/file/d/1ek7vxhe6grPAlwgxl3wJMbZibaKuNcdq/view?usp=sharing) and [objbg](https://drive.google.com/file/d/1cbfnXJh7KYfldrIMTbTNF_J1xukrosRu/view?usp=sharing).

The randomness is caused by random seeds. Even under the same random seed, results are different on different CUDA devices. See https://github.com/pytorch/pytorch/issues/18412, https://github.com/pytorch/pytorch/issues/25690.

**Comment by cuge1995** [comment]
> Hi,
> 
> We have revised the reported result of scan_objonly and scan-objbg. Scan_objonly acc is 88.29% and scan-objbg acc is 90.02%.
> 
> We double-check the experiments, and can easily get reported results. See new log [objonly](https://drive.google.com/file/d/1ek7vxhe6grPAlwgxl3wJMbZibaKuNcdq/view?usp=sharing) and [objbg](https://drive.google.com/file/d/1cbfnXJh7KYfldrIMTbTNF_J1xukrosRu/view?usp=sharing).
> 
> The randomness is caused by random seeds. Even under the same random seed, results are different on different CUDA devices. See [pytorch/pytorch#18412](https://github.com/pytorch/pytorch/issues/18412), [pytorch/pytorch#25690](https://github.com/pytorch/pytorch/issues/25690).

It seems that the results are different in Table 1 in your paper where OBJ-BJ is 88.29 and OBJ-ONLY 90.01.
